.github/workflows/sync.yml

```diff
@@ -0,0 +1,23 @@
+name: Sync policies
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      # https://github.com/marketplace/actions/repo-file-sync-action
+      - name: Run GitHub File Sync
+        uses: BetaHuhn/repo-file-sync-action@v1.20.0
+        with:
+          GH_PAT: ${{ secrets.FLOWZONE_TOKEN }}
```

Comment:
Do we want to reuse this token or get a new one?

Reply:
@klutchell as soon as `balena team` user accepts the invite, it should start working: https://github.com/orgs/product-os-test/people/pending_invitations

Reply:
all of our `FLOWZONE_TOKEN`s are issued under that user, (e.g.):

```
$ gh api   -H "Accept: application/vnd.github+json"   /user
{
  "login": "balenaio",
  "id": 13831732,
  "node_id": "MDQ6VXNlcjEzODMxNzMy",
  "avatar_url": "https://avatars.githubusercontent.com/u/13831732?v=4",
  "gravatar_id": "",
  "url": "https://api.github.com/users/balenaio",
  "html_url": "https://github.com/balenaio",
  "followers_url": "https://api.github.com/users/balenaio/followers",
  "following_url": "https://api.github.com/users/balenaio/following{/other_user}",
  "gists_url": "https://api.github.com/users/balenaio/gists{/gist_id}",
  "starred_url": "https://api.github.com/users/balenaio/starred{/owner}{/repo}",
  "subscriptions_url": "https://api.github.com/users/balenaio/subscriptions",
  "organizations_url": "https://api.github.com/users/balenaio/orgs",
  "repos_url": "https://api.github.com/users/balenaio/repos",
  "events_url": "https://api.github.com/users/balenaio/events{/privacy}",
  "received_events_url": "https://api.github.com/users/balenaio/received_events",
  "type": "User",
  "site_admin": false,
  "name": "Balena team",
  "company": "@balena-io ",
  "blog": "https://balena.io/",
  "location": null,
  "email": null,
  "hireable": null,
  "bio": "Balena.io hackers, see @balena-io for our repositories!",
  "twitter_username": null,
  "public_repos": 0,
  "public_gists": 0,
  "followers": 7,
  "following": 0,
  "created_at": "2015-08-17T10:40:26Z",
  "updated_at": "2023-01-09T08:08:14Z"
}

```